blackduck/Client.py

```diff
@@ -56,24 +56,33 @@ def __init__(self, base_url, timeout, retries, verify):
 
     def request(self, method, url, **kwargs):
         kwargs['timeout'] = self._timeout
+        if method.lower() == 'get':
+            # set default media type if not provided
+            headers = {
+                'accept': "application/json",
+                'content-type': "application/json"
+            }
+            headers.update(kwargs.pop('headers', dict()))
+            kwargs['headers'] = headers
```

Comment:
Putting this here allows the user to get the benefit of the default headers when invoking direct URLs e.g. bd.session.get("/api/projects?offset=0&limit=5")